test/common/policy_utils.go

```diff
@@ -69,6 +69,99 @@ func PatchPlacementRule(namespace, name string) error {
 	return err
 }
 
+// ApplyPlacement function creates Placement and PlacementBinding so that it will
+// always only match the targetCluster.
+func ApplyPlacement(ctx SpecContext, namespace, policyName string) error {
+	By("Apply Placement and PlacementBinding" + namespace + "/" +
+		"placement-" + policyName + "/" + "placement-binding-" + "policyName" +
+		" with clusterSelector {name: " + ClusterNamespaceOnHub + "}")
+
+	placement := unstructured.Unstructured{
+		Object: map[string]interface{}{
+			"apiVersion": GvrPlacement.Group + "/" + GvrPlacement.Version,
+			"kind":       "Placement",
+			"metadata": map[string]interface{}{
+				"name": "placement-" + policyName,
+			},
+			"spec": map[string]interface{}{
+				"predicates": []interface{}{
+					map[string]interface{}{
+						"requiredClusterSelector": map[string]interface{}{
+							"labelSelector": map[string]interface{}{
+								"matchExpressions": []interface{}{
+									map[string]interface{}{
+										"key":      "name",
+										"operator": "In",
+										"values": []string{
+											ClusterNamespaceOnHub,
+										},
+									},
+								},
+							},
+						},
+					},
+				},
+			},
+		},
```

Comment:
(Optional) This would be more compact
```suggestion
										}}}}}}}}},
```